quarkus-logs-manager/run.sh

```
@@ -0,0 +1,3 @@
+docker-compose -p quarkus-logs-manager up -d postgres
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)